Reports/progress_report.md

```diff
@@ -0,0 +1,296 @@
+# Data Safe Haven project report
+
+This document contains a summary of progress across all stories in the [project roadmap](https://github.com/orgs/alan-turing-institute/projects/111/views/1).
+It maps stories according to the (main) pillar and priority they contribute to.
+
+Included stories are those that were scheduled and/or prioritised over this period (February 2023-February 2024), some not scheduled stories are included when there are updates for them regardless (indirect contirbutions from other stories, relevant work recently started, or something to report in general).
+
+## Infrastructure as code
+Running projects working with sensitive data safely
+Running cutting edge data science projects effectively
+
+
+### Manage codebase releases and testing
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/50
+
+Contributes to:
+- Running projects working with sensitive data safely
+- Running cutting edge data science projects effectively
+
+#### Goal
+Support for deployments of the Data Safe Haven at Turing and beyond
+
+#### Progress
+- Pen testing done: little found
+- Penetration tested arranged and will be done in September
+- Preparation for release v4.1.0: Deployment of different SRE variants, Security checklist
+- Reviewing v4.1.0:  No significant problems is deployment logs, Problems found in security checklist relating to MSRDS
+- Working on [Release 4.1.0](https://github.com/alan-turing-institute/data-safe-haven/issues/1544): fixes bugs and introduces necessary updates 
```

Comment:
Part of the confusion here I think is that I finished this too late for the team to come in and give a more comprenhensive overview of progress and where we are at.

The story issue is open because it refers to ongoing work to keep making releases, and we are working on 4.2

That warrants the questions on wheter to have ongoing Stories or more time bound ones, and also we need to better explain what goes into which story as work towards 4.2 is happening in Codebase Maintenance. This story deals with the release itself.

And yes, happy to reverse order/clarify timeline.